app/assets/javascripts/index/directions.js

```diff
@@ -127,8 +146,6 @@ OSM.Directions = function (map) {
     };
 
     endpoint.setLatLng = function (ll) {
-      var precision = OSM.zoomPrecision(map.getZoom());
-      input.val(ll.lat.toFixed(precision) + ", " + ll.lng.toFixed(precision));
```

Comment:
This code was used by `$("#map").on("drop" ...` to populate an input when the marker next to the input is dragged and dropped on the map.

What happens now:
[Screencast from 04.07.2024 06:35:02.webm](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/9d85463c-aeb6-4007-87f9-32396d413cf7)

What used to happen:
[Screencast from 04.07.2024 06:38:26.webm](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/16a35160-cc5b-46f3-acad-ccfcb9c8c0b0)